hack/common.sh

```diff
@@ -544,16 +544,18 @@ os::build::require_clean_tree() {
 # used to determine the common range with the second argument. If the first argument
 # is not an integer, it is assumed to be a Git commit range and output directly.
 os::build::commit_range() {
+  local remote
+  remote="$(git remote -v | grep kubernetes/kubernetes.git | awk '{print $1}' | uniq)"
```

Comment:
This won't work - it assume commit range is over Kube which is not the case.  You can support an optional second argument or env var override.